finality-aleph/src/new_network/connection_manager/session.rs

```diff
@@ -88,6 +88,14 @@ impl Handler {
         self.own_authentication.clone()
     }
 
+    /// Returns the authentication for the node with the given index, if we have it.
+    pub fn authentication_for(&self, node_id: &NodeIndex) -> Option<Authentication> {
```

Comment:
for completeness, I would add a testcase presenting what this method returns for own id

Reply:
Added.